src/compiler/commandLineParser.ts

```diff
@@ -379,6 +378,68 @@ namespace ts {
     }
 
     /**
+     * Parses non quoted strings separated by comma e.g. "a,b" would result in string array ["a", "b"]
+     * @param s
+     * @param existingValue
+     */
+    function parseMultiValueStringArray(s: string, existingValue: string[]) {
+        let value: string[] = existingValue || [];

```

Comment:
You shouldn't need the type annotation; the right side will be contextually typed by the left side.